tests/regression/test_bcs.py

```diff
@@ -1,403 +0,0 @@
-import pytest
```

Comment:
I renamed it `test_to_expression.py` because it's not a test on BCs.
DirichletBCs calls the function `expression.to_expression`, and this
is the function I modified, and that should therefore be tested.